### PR TITLE
Fix the "readfile" to include the file's TOC

### PR DESCRIPTION
**Pull request by RichieEscarez**
enable rightside nav TOC for any file that is "included" using this readfile shortcode

### CODE REVIEW

File: layouts/shortcodes/readfile.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
   {{ $.Scratch.Set "filepath" ( .Get "file" ) }}
 {{ end }}
 {{ if eq (.Get "markdown") "true" }}
-  {{ $.Scratch.Get "filepath" | readFile | markdownify }}
+  {{ readFile ($.Scratch.Get "filepath") }}
```
Comment (evankanderson) [inline layouts/shortcodes/readfile.md]: Did you mean to lose the `markdownify`?

Comment (RichieEscarez) [reply layouts/shortcodes/readfile.md]: Yes, Im a part of the Docsy open source team now and Hugo recently fixed how it processes shortcodes. For shortcodes called with {{% .... %}} syntax that use %, they all get sent to the markdown processor (blackfriday) so `markdownify` is not needed.

The improvement from the PR is shown here:
- https://deploy-preview-53--knative.netlify.com/docs/

Vs our currently published version that excludes the left-side "in-page" nav tree: https://knative.dev/docs/

In line with our other PR about code, if we call {{< ... >}} the readfile shortcode using <, i think it just copies in the targeted file but I need to test and verify.

**Comment by RichieEscarez** [comment]
FYI, im going to start testing the readfile for use with code samples. Can someone please take a look so we can merge this before I need to make any additional changes for handling non-markdown/html files?

**Pull request merged**